dashboards/Gamestudio-qs/config.yml

```
@@ -0,0 +1,29 @@
+# Name of the pack (required)
+name: GameStudioQS
```

We don't include the `quickstart` wording in the name, so this can be simplified and let's get these fields updated
```suggestion
slug: gamestudio
title: Game Studio
```